cypress.config.ts

```diff
@@ -28,11 +28,11 @@ export default defineConfig({
             return config;
         },
         baseUrl: "http://localhost:3200",
-        video: false,
+        video: true,
         videoUploadOnPasses: false,
-        screenshotOnRunFailure: false,
+        screenshotOnRunFailure: true,
+        videoCompression: 1,
```

Comment:
did you figure out how to set a really small retention period? (e.g. delete after a day)

Reply:
yep, it's what the `retention-days` key does in `checks.yml`